### PR TITLE
Get prevalence and ART coverage for uploaded ANC data

### PR DESCRIPTION
**Pull request by r-ash**
This PR will

* Return prevalence and ART coverage from uploaded ANC data for use in plots

### CODE REVIEW

File: tests/testthat/test-validate-inputs.R [hunk 1]
```diff
@@ -86,13 +86,15 @@ test_that("do_validate_programme validates programme file", {
   expect_equal(data$filters$age, expected_filters)
 })
 
-test_that("do_validate_anc validates ANC file", {
+test_that("do_validate_anc validates ANC file and gets data for plotting", {
```
Comment (richfitz) [inline tests/testthat/test-validate-inputs.R]: Here you test for existance of `prevalance` and `art_coverage` but in the schema only for `area_id` (`AncResponseData.schema.json`). Should these be harmonised?

File: R/validate_inputs.R [hunk 2]
```diff
@@ -152,9 +152,7 @@ do_validate_anc <- function(anc, shape) {
   assert_consistent_regions(read_regions(shape, "shape"),
                             read_regions(anc, "anc"),
                             "ANC")
-  ## TODO: Call naomi to calculate prevalence & art_coverage which is what we
-  ## need to return to the front end for plotting. mrc-492
-  list(data = data,
+  list(data = naomi::calculate_prevalence_art_coverage(data),
```
Comment (richfitz) [inline R/validate_inputs.R]: Please update the DESCRIPTION to depend on `naomi (>= 0.0.2)`

File: tests/testthat/test-validate-inputs.R (hunk 1 above)
Comment (r-ash) [reply tests/testthat/test-validate-inputs.R]: Yeah they should, I've updated that now.

File: R/validate_inputs.R (hunk 2 above)
Comment (r-ash) [reply R/validate_inputs.R]: Thanks, updated that.

**Pull request merged**